functions/zoom-downloader.py

```diff
@@ -565,41 +575,52 @@ def stream_file_to_s3(self):
             {"uploading file to S3": self.s3_filename,
              "metadata": metadata}
         )
-        part_info = {"Parts": []}
-        mpu = s3.create_multipart_upload(
+        parts = []
+        mpu = self.s3.create_multipart_upload(
                     Bucket=ZOOM_VIDEOS_BUCKET,
                     Key=self.s3_filename,
                     Metadata=metadata)
+        self.upload_id = mpu["UploadId"]
```

Comment:
since the object/class doesn't use this value anywhere else i wonder if it shouldn't just be another arg to `upload_part`?